From a8ff1e964b2b8cd0b60362c76bd92795cee6b3c3 Mon Sep 17 00:00:00 2001
From: Kamil Dudka <kdudka@redhat.com>
Date: Sun, 17 Apr 2016 22:36:13 +0200
Subject: [PATCH] doc: clarify exit status handling of -exec command {} +

* find/find.1 (-exec): Explain how exit status is propagated if the
-exec command {} + syntax is used.
(-execdir): Likewise.

Reported at https://bugzilla.redhat.com/1325049

Upstream-commit: ae424b959c5e9bd23f9f686cb34653bc4cd1270e
Signed-off-by: Kamil Dudka <kdudka@redhat.com>
---
 find/find.1 | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/find/find.1 b/find/find.1
index a36a0bc..c4aaf17 100644
--- a/find/find.1
+++ b/find/find.1
@@ -1069,6 +1069,9 @@ command line is built in much the same way that
 .B xargs
 builds its command lines.  Only one instance of `{}' is allowed within
 the command.  The command is executed in the starting directory.  If
+any invocation returns a non-zero value as exit status, then
+.B find
+returns a non-zero exit status.  If
 .B find
 encounters an error, this can sometimes cause an
 immediate exit, so some pending commands may not be run
@@ -1104,6 +1107,9 @@ appropriately-named file in a directory in which you will run
 The same applies to having entries in
 .B $PATH
 which are empty or which are not absolute directory names.  If
+any invocation returns a non-zero value as exit status, then
+.B find
+returns a non-zero exit status.  If
 .B find
 encounters an error, this can sometimes cause an
 immediate exit, so some pending commands may not be run
-- 
2.5.5

